lib/eslint.js

```diff
@@ -144,6 +145,7 @@ function addDeclaredGlobals(program, globalScope, config) {
         }
     });
 
+    assign(exportedGlobals, config.exported);
```

Comment:
I'm slightly concern about ByRef issues with this. If exportedGlobals will be modified later (which it will be), those changes will be carried over to config.exported as well, and will most likely affect other files. We've had a lot of issues with shallow copy lately, this seems like it could be another one at some point.